ste/mgr-JobMgr.go

```diff
@@ -231,6 +256,39 @@ func (jm *jobMgr) logPerfInfo(displayStrings []string, constraint common.PerfCon
 	jm.Log(pipeline.LogInfo, msg)
 }
 
+// TODO: find a better way for JobsAdmin to log (it doesn't have direct access to the job log, because it was originally designed to support multilpe jobs
```

Comment:
Perhaps if/when we get around to refactoring this general area of code we can unify jobsAdmin and jobMgr

Reply:
That's an interesting thought.... Let's keep it in mind.